src/components/toast/global_toast_list.tsx

```diff
@@ -283,6 +291,28 @@ export const EuiGlobalToastList: FunctionComponent<EuiGlobalToastListProps> = ({
     );
   });
 
+  if (allowClearAll && toasts.length > 1) {
+    const dismissAllToastImmediately = () => {
+      toasts.forEach((toast) => dismissToastProp(toast));
+      dismissAllToastsCallback?.();
+    };
+
+    renderedToasts.push(
+      <EuiGlobalToastListItem
+        data-test-subj="clear-all-toast"
```

Comment:
We use camelCasing for our data-test-subj strings, and generally try to use an EUI namespace (although this is not terribly consistent at times)
```suggestion
        data-test-subj="euiClearAllToastsButton"
```

Reply:
Thanks for pointing this out!